src/front/app/components/Create/create-header.scss

```diff
@@ -0,0 +1,58 @@
+.challenge-create {
+
+    &__header {
+        margin-left: auto;
+        margin-right: auto;
+        background: $create-header-color;
```

Comment:
I am not sure we really need variable for every color (or whatever) - I mean it make sense to use variables for some value if it used at least 2-3 times in code